Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:latest
+MAINTAINER Barak Michener <me@barakmich.com>
+
+# Set up workdir
+WORKDIR      /go/src/github.com/google/cayley
+RUN mkdir -p /go/src/github.com/google/cayley
+
+# Restore vendored dependencies
+RUN go get github.com/tools/godep
+ADD Godeps /go/src/github.com/google/cayley/Godeps
+RUN godep restore
+
+# Add and install cayley
+ADD . .
+RUN go install github.com/google/cayley/cmd/cayley
```

Comment:
I've added -v, but you cannot use "./..." without getting a failure because the `integration` directory contains no buildable source files.